### PR TITLE
run_if_changed for azure-file-windows-containerd test

### PR DESCRIPTION
**Pull request by andyzhangx**
cc @ravisantoshgudimetla

### CODE REVIEW

**Comment by andyzhangx** [comment]
```
...ERROR:root:webhook failed signature check
.ERROR:root:webhook failed signature check
........................ERROR:root:unable to fetch 'http://testbed.example.com/_ah/gcs/foo/quux': status code 404
....ERROR:root:failed to get user name: status 403
.ERROR:root:failed to vend token: status 403
```

File: config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml [hunk 1]
```diff
@@ -381,7 +381,7 @@ presubmits:
     decorate: true
     decoration_config:
       timeout: 2h
-    always_run: false
+    always_run: true
```
Comment (BenTheElder) [inline config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml:384]: /hold
Please don't add new presubmits to always run in kubernetes/kubernetes without discussion.

Comment (andyzhangx) [reply config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml]: here is the original tracking issue: https://github.com/kubernetes/kubernetes/issues/97304, I changed the PR a little ro `run_if_changed: 'azure.*\.go$|.*windows\.go$'`
@BenTheElder

Comment (andyzhangx) [reply config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml]: is that ok to merge this PR since there is already discussion here: https://github.com/kubernetes/kubernetes/issues/97304
The azure file test on windows containerd is stable now, we want to make it if there is windows code change, thanks.

Comment (spiffxp) [reply config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml]: We are behind on documenting policy for this, so here's my take as a SIG Testing chair.

Adding a new merge-blocking job needs careful consideration (ref: https://github.com/kubernetes/test-infra/issues/18729)
- Merge-blocking jobs are much more expensive to run given the volume of PR traffic we receive
- More jobs-per-PR means higher flakes-per-PR if we assume each job has some non-zero chance of flaking

We also want to ensure that merge-blocking coverage is a subset of release-blocking coverage.  That is:
- if there are tests that are not release-blocking, they should not be merge-blocking
- If there are ~(non-esoteric, non-expensive, high-signal) tests that are part of release-blocking, we should aspire to run them as part of merge-blocking to get earlier signal on whether changes are likely to break the release

The reason we allowed ourselves the extra jobs while switching from dockershim to containerd for the jobs mentioned in https://github.com/kubernetes/kubernetes/issues/97304 (gce, scalability) is that we were replacing one merge-blocking job for another.

tl;dr if there is release-blocking coverage, and the job cost isn't too egregious, then I'm probably ok with run_if_changed for now

**Review by spiffxp — CHANGES_REQUESTED** [review]
Given that is running `--test-azure-file-csi-driver`, are you sure this is the appropriate set of files to trigger on?  Is there some way to narrow this down?
```
$ find . | grep -E 'azure.*\.go$|.*windows\.go$' | grep -v './vendor' | sort
./cluster/images/etcd/migrate/utils_windows.go
./cmd/kube-proxy/app/init_windows.go
./cmd/kube-proxy/app/server_windows.go
./cmd/kubeadm/app/apis/kubeadm/v1beta1/defaults_windows.go
./cmd/kubeadm/app/apis/kubeadm/v1beta2/defaults_windows.go
./cmd/kubeadm/app/constants/constants_windows.go
./cmd/kubeadm/app/images/images_windows.go
./cmd/kubeadm/app/preflight/checks_windows.go
./cmd/kubeadm/app/util/chroot_windows.go
./cmd/kubeadm/app/util/initsystem/initsystem_windows.go
./cmd/kubeadm/app/util/runtime/runtime_windows.go
./cmd/kubelet/app/init_windows.go
./cmd/kubelet/app/options/osflags_windows.go
./pkg/credentialprovider/azure/azure_acr_helper.go
./pkg/credentialprovider/azure/azure_credentials.go
./pkg/credentialprovider/azure/azure_credentials_test.go
./pkg/credentialprovider/azure/doc.go
./pkg/kubelet/apis/config/validation/validation_windows.go
./pkg/kubelet/cadvisor/cadvisor_windows.go
./pkg/kubelet/cm/container_manager_windows.go
./pkg/kubelet/cm/internal_container_lifecycle_windows.go
./pkg/kubelet/cri/remote/fake/endpoint_windows.go
./pkg/kubelet/cri/remote/util/util_windows.go
./pkg/kubelet/dockershim/cm/container_manager_windows.go
./pkg/kubelet/dockershim/docker_container_windows.go
./pkg/kubelet/dockershim/docker_image_windows.go
./pkg/kubelet/dockershim/docker_stats_windows.go
./pkg/kubelet/dockershim/docker_streaming_windows.go
./pkg/kubelet/dockershim/helpers_windows.go
./pkg/kubelet/dockershim/network/cni/cni_windows.go
./pkg/kubelet/kubelet_node_status_windows.go
./pkg/kubelet/kuberuntime/kuberuntime_container_windows.go
./pkg/kubelet/kuberuntime/security_context_windows.go
./pkg/kubelet/server/stats/summary_sys_containers_windows.go
./pkg/kubelet/stats/cri_stats_provider_windows.go
./pkg/kubelet/util/util_windows.go
./pkg/proxy/userspace/rlimit_windows.go
./pkg/scheduler/internal/cache/debugger/signal_windows.go
./pkg/volume/awsebs/attacher_windows.go
./pkg/volume/azure_file/azure_file.go
./pkg/volume/azure_file/azure_file_test.go
./pkg/volume/azure_file/azure_provision.go
./pkg/volume/azure_file/azure_util.go
./pkg/volume/azure_file/doc.go
./pkg/volume/azuredd/attacher.go
./pkg/volume/azuredd/attacher_test.go
./pkg/volume/azuredd/azure_common.go
./pkg/volume/azuredd/azure_common_linux.go
./pkg/volume/azuredd/azure_common_test.go
./pkg/volume/azuredd/azure_common_unsupported.go
./pkg/volume/azuredd/azure_common_windows.go
./pkg/volume/azuredd/azure_dd.go
./pkg/volume/azuredd/azure_dd_block.go
./pkg/volume/azuredd/azure_dd_block_test.go
./pkg/volume/azuredd/azure_dd_max_disk_count.go
./pkg/volume/azuredd/azure_dd_test.go
./pkg/volume/azuredd/azure_mounter.go
./pkg/volume/azuredd/azure_provision.go
./pkg/volume/azuredd/azure_provision_test.go
./pkg/volume/azuredd/doc.go
./pkg/volume/util/fs/fs_windows.go
./pkg/volume/util/hostutil/hostutil_windows.go
./pkg/volume/util/subpath/subpath_windows.go
./pkg/volume/vsphere_volume/vsphere_volume_util_windows.go
./staging/src/k8s.io/apiserver/pkg/server/options/serving_windows.go
./staging/src/k8s.io/apiserver/pkg/server/signal_windows.go
./staging/src/k8s.io/client-go/plugin/pkg/client/auth/azure/azure.go
./staging/src/k8s.io/client-go/plugin/pkg/client/auth/azure/azure_test.go
./staging/src/k8s.io/csi-translation-lib/plugins/azure_disk.go
./staging/src/k8s.io/csi-translation-lib/plugins/azure_disk_test.go
./staging/src/k8s.io/csi-translation-lib/plugins/azure_file.go
./staging/src/k8s.io/csi-translation-lib/plugins/azure_file_test.go
./staging/src/k8s.io/kubectl/pkg/util/term/resizeevents_windows.go
./staging/src/k8s.io/kubectl/pkg/util/umask_windows.go
./staging/src/k8s.io/legacy-cloud-providers/azure/auth/azure_auth.go
./staging/src/k8s.io/legacy-cloud-providers/azure/auth/azure_auth_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/auth/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_backoff.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_backoff_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_blobDiskController.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_blobDiskController_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_config.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_config_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_common.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_common_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_standard.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_standard_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_vmss.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_vmss_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_fakes.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_file.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_instance_metadata.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_instances.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_instances_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_managedDiskController.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_managedDiskController_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_ratelimit.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_ratelimit_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_routes.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_routes_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_standard.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_standard_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_storage.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_storage_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_storageaccount.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_storageaccount_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_utils.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_utils_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_vmsets.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_vmss.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_vmss_cache.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_vmss_cache_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_vmss_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_wrap.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_wrap_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_zones.go
./staging/src/k8s.io/legacy-cloud-providers/azure/azure_zones_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/cache/azure_cache.go
./staging/src/k8s.io/legacy-cloud-providers/azure/cache/azure_cache_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/cache/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/azure_armclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/azure_armclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/mockarmclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/mockarmclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/azure_client_config.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/azure_client_config_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/containerserviceclient/azure_containerserviceclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/containerserviceclient/azure_containerserviceclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/containerserviceclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/containerserviceclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/containerserviceclient/mockcontainerserviceclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/containerserviceclient/mockcontainerserviceclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/deploymentclient/azure_deploymentclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/deploymentclient/azure_deploymentclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/deploymentclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/deploymentclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/deploymentclient/mockdeploymentclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/deploymentclient/mockdeploymentclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/diskclient/azure_diskclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/diskclient/azure_diskclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/diskclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/diskclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/diskclient/mockdiskclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/diskclient/mockdiskclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/fileclient/azure_fileclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/fileclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/fileclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/fileclient/mockfileclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/fileclient/mockfileclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/interfaceclient/azure_interfaceclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/interfaceclient/azure_interfaceclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/interfaceclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/interfaceclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/interfaceclient/mockinterfaceclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/interfaceclient/mockinterfaceclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/loadbalancerclient/azure_loadbalancerclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/loadbalancerclient/azure_loadbalancerclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/loadbalancerclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/loadbalancerclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/loadbalancerclient/mockloadbalancerclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/loadbalancerclient/mockloadbalancerclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/publicipclient/azure_publicipclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/publicipclient/azure_publicipclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/publicipclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/publicipclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/publicipclient/mockpublicipclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/publicipclient/mockpublicipclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/routeclient/azure_routeclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/routeclient/azure_routeclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/routeclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/routeclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/routeclient/mockrouteclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/routeclient/mockrouteclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/routetableclient/azure_routetableclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/routetableclient/azure_routetableclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/routetableclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/routetableclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/routetableclient/mockroutetableclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/routetableclient/mockroutetableclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/securitygroupclient/azure_securitygroupclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/securitygroupclient/azure_securitygroupclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/securitygroupclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/securitygroupclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/securitygroupclient/mocksecuritygroupclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/securitygroupclient/mocksecuritygroupclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/snapshotclient/azure_snapshotclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/snapshotclient/azure_snapshotclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/snapshotclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/snapshotclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/snapshotclient/mocksnapshotclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/snapshotclient/mocksnapshotclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/mockstorageaccountclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/mockstorageaccountclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/subnetclient/azure_subnetclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/subnetclient/azure_subnetclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/subnetclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/subnetclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/subnetclient/mocksubnetclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/subnetclient/mocksubnetclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmclient/azure_vmclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmclient/azure_vmclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmclient/mockvmclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmclient/mockvmclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmsizeclient/azure_vmsizeclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmsizeclient/azure_vmsizeclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmsizeclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmsizeclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmsizeclient/mockvmsizeclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmsizeclient/mockvmsizeclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmssclient/azure_vmssclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmssclient/azure_vmssclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmssclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmssclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmssclient/mockvmssclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmssclient/mockvmssclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmssvmclient/azure_vmssvmclient.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmssvmclient/azure_vmssvmclient_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmssvmclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmssvmclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmssvmclient/mockvmssvmclient/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmssvmclient/mockvmssvmclient/interface.go
./staging/src/k8s.io/legacy-cloud-providers/azure/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/main_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/metrics/azure_metrics.go
./staging/src/k8s.io/legacy-cloud-providers/azure/metrics/azure_metrics_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/metrics/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/mockvmsets/azure_mock_vmsets.go
./staging/src/k8s.io/legacy-cloud-providers/azure/mockvmsets/doc.go
./staging/src/k8s.io/legacy-cloud-providers/azure/retry/azure_error.go
./staging/src/k8s.io/legacy-cloud-providers/azure/retry/azure_error_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/retry/azure_retry.go
./staging/src/k8s.io/legacy-cloud-providers/azure/retry/azure_retry_test.go
./staging/src/k8s.io/legacy-cloud-providers/azure/retry/doc.go
./staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere_util_windows.go
./staging/src/k8s.io/mount-utils/mount_helper_windows.go
./staging/src/k8s.io/mount-utils/mount_windows.go
./staging/src/k8s.io/sample-controller/pkg/signals/signal_windows.go
./test/e2e/framework/providers/azure/azure.go
./test/images/agnhost/dns/dns_windows.go
./test/images/agnhost/mounttest/mt_utils_windows.go
```

**Comment by andyzhangx** [comment]
> Given that is running `--test-azure-file-csi-driver`, are you sure this is the appropriate set of files to trigger on? Is there some way to narrow this down?
> 
> ```
> $ find . | grep -E 'azure.*\.go$|.*windows\.go$' | grep -v './vendor' | sort
> ./cluster/images/etcd/migrate/utils_windows.go
> ./staging/src/k8s.io/sample-controller/pkg/signals/signal_windows.go
> ./test/e2e/framework/providers/azure/azure.go
> ./test/images/agnhost/dns/dns_windows.go
> ./test/images/agnhost/mounttest/mt_utils_windows.go
> ```

@spiffxp the test cases are in `azure-file-csi-driver` repo, and it actually could test against both in-tree and CSI azure file drivers, you could see we already have in-tree azure file driver e2e test against docker node:
https://github.com/kubernetes/test-infra/blob/97361e18137f1d51fbb19f006d41341c2c1a83fa/config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml#L200-L221

File: config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml (hunk 1 above)
Comment (andyzhangx) [reply config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml]: thanks for clarification, it's not blocking test since it's optional test.
/hold cancel

**Review by ZeroMagic — APPROVED** [review]
/lgtm
